src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +219,19 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+
+.visEditorSidebar__aggGroup .euiAccordion {
+  // to apply truncated string with ellipsis for agg description
+  & > .euiFlexGroup > .euiFlexItem:first-child,
+  .euiAccordion__buttonContent,
```

Comment:
To make an agg description truncated, it's needed to apply some styles.

Before | After |
----|----
<img width="442" alt="truncate-no" src="https://user-images.githubusercontent.com/11524637/61465376-9f98a580-a980-11e9-9422-3e8c509457ce.png">|<img width="407" alt="truncate" src="https://user-images.githubusercontent.com/11524637/61465350-93ace380-a980-11e9-880a-b16c3e7c0122.png">

Reply:
It was fixed in https://github.com/elastic/eui/pull/2143.